Bobsar0/Fast-Food-Fast
# #161107840 Improve test coverage

#### What does this PR do?
Improves test coverage
#### Description of Task to be completed?
- Write more tests for each endpoint
- Modify implementation of endpoints to pass tests 

#### How should this be manually tested?
  Clone the  repo and run the following commands:
``` 
  git checkout ch-improve-tests-#161107840
  npm test
 ```
You should see all test cases with detailed overall test statistics for all files.

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/ #161107840
